provider/datagen/src/transform/cldr/datetime/symbols.rs

```diff
@@ -64,7 +64,7 @@ fn get_month_code_map(calendar: &str) -> &'static [TinyStr4] {
         "gregory" | "buddhist" | "japanese" | "japanext" | "indian" | "persian" | "roc" => {
             &SOLAR_MONTH_CODES[0..12]
         }
-        "coptic" | "ethiopic" => SOLAR_MONTH_CODES,
+        "coptic" | "ethiopic" | "chinese" => SOLAR_MONTH_CODES,
```

Comment:
Please add a TODO comment here with the issue number for fixing leap month formatting